knowledge-base/missing-symbols-in-pdf.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Missing symbols in PdfProcessing
+description:  Missing symbols in PdfProcessing
+type: how-to troubleshooting
+page_title:  Missing symbols in PdfProcessing
+slug: missing-symbols-in-pdf
+position: 0
+tags: pdf, umlaut, symbol
+res_type: kb
+---
+
+|Product Version|Product|Author|
+|----|----|----|
+|2020.1.218|PdfProcessing|[Dimitar Karamfilov](https://www.telerik.com/blogs/author/dimitar-karamfilov)|
+
+## Problem 
+This can happen when the characters cannot be found in a specific font. In this case, we either fall back to another font or draw nothing. 
```

Comment:
we either fall back to another font or draw nothing - the PdfProcessing either fall back to...